packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -1974,6 +1974,135 @@ void main() {
     expect(itemMaterial.color, themeData.colorScheme.onSurface.withOpacity(0.12));
   });
 
+  testWidgets(
+    'When the initial selection matches a menu entry the text field should display the corresponding value',
```

Comment:
```suggestion
    'When the initial selection matches a menu entry, the text field displays the corresponding value',
```

Just a personal preference – I like going for "declarative vibes" 😎